pages/validator/external-chains/avalanche.md

```diff
@@ -1,76 +1,189 @@
 
 ```bash
-curl -X POST --data '{"jsonrpc": "2.0","method": "info.isBootstrapped","params":{"chain":"C"},"id":1}' -H 'content-type:application/json;' localhost:9650/ext/info
+# change log settings to persistent if not already
+sed -i 's/#Storage=auto/Storage=persistent/g' /etc/systemd/journald.conf
+sudo systemctl restart systemd-journald
 
+journalctl -u avalanchego.service -f -n 100 -o cat
+```
+Logs should appear like this
+```bash
+[11-28|07:50:59.280] INFO <P Chain> bootstrap/bootstrapper.go:481 fetching blocks {"numFetchedBlocks": 85000, "numTotalBlocks": 3241631, "eta": "1h52m8s"}
+[11-28|07:51:13.370] INFO <P Chain> bootstrap/bootstrapper.go:481 fetching blocks {"numFetchedBlocks": 90000, "numTotalBlocks": 3241631, "eta": "1h53m58s"}
+[11-28|07:51:29.644] INFO <P Chain> bootstrap/bootstrapper.go:481 fetching blocks {"numFetchedBlocks": 95000, "numTotalBlocks": 3241631, "eta": "1h56m46s"}
+[11-28|07:51:43.864] INFO <P Chain> bootstrap/bootstrapper.go:481 fetching blocks {"numFetchedBlocks": 100000, "numTotalBlocks": 3241631, "eta": "1h58m12s"}
+[11-28|07:51:52.789] INFO <P Chain> bootstrap/bootstrapper.go:481 fetching blocks {"numFetchedBlocks": 105000, "numTotalBlocks": 3241631, "eta": "1h56m50s"}
+[11-28|07:52:06.814] INFO <P Chain> bootstrap/bootstrapper.go:481 fetching blocks {"numFetchedBlocks": 110000, "numTotalBlocks": 3241631, "eta": "1h58m0s"}
+[11-28|07:52:17.979] INFO <P Chain> bootstrap/bootstrapper.go:481 fetching blocks {"numFetchedBlocks": 115000, "numTotalBlocks": 3241631, "eta": "1h57m45s"}
+[11-28|07:52:31.488] INFO <P Chain> bootstrap/bootstrapper.go:481 fetching blocks {"numFetchedBlocks": 120000, "numTotalBlocks": 3241631, "eta": "1h58m31s"}
+[11-28|07:52:40.475] INFO <P Chain> bootstrap/bootstrapper.go:481 fetching blocks {"numFetchedBlocks": 125000, "numTotalBlocks": 3241631, "eta": "1h57m20s"}
+[11-28|07:52:53.462] INFO <P Chain> bootstrap/bootstrapper.go:481 fetching blocks {"numFetchedBlocks": 130000, "numTotalBlocks": 3241631, "eta": "1h57m49s"}
 ```
 
-If the node is successfully synced, the output from above will print `{"jsonrpc":"2.0","result":{"isBootstrapped":true},"id":1}`
+# Verify
 
-#### EVM RPC endpoint URL
+```bash
+YOUR_IP=$(curl -4 ifconfig.co)
+
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"info.isBootstrapped",
+    "params": {
+        "chain":"X"
+    }
+}' -H 'content-type:application/json;' $YOUR_IP:9650/ext/info | jq
+```
 
-Axelar Network will be connecting to the C-Chain, which is the EVM compatible blockchain, so your `rpc_addr` should be exposed in this format:
+If you get something like this in response of the above rpc call, your node is setup correctly
 
+Wait for `isBootstrapped` to become true before using it in vald config
 ```bash
-http://IP:PORT/ext/bc/C/rpc
+{
+  "jsonrpc": "2.0",
+  "result": {
+    "isBootstrapped": false # wait for it to become true
+  },
+  "id": 1
+}
 ```
 
-Example:
-`http://192.168.192.168:9650/ext/bc/C/rpc`
+## Endpoint
```

Comment:
can you also add the vald config for it, similar to what we do for Ethereum?

Reply:
Sure